src/GrassPokemon.java

```diff
@@ -0,0 +1,156 @@
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+public class GrassPokemon extends Pokemon {
+    private final String name;
+    private int hp;
+    private int level;
+    private final String food;
+    private final String sound;
+    private final List<String> attacks;
+
+
+    public GrassPokemon(String name, int hp, int level, String food, String sound) {
+        this.name = name;
+        this.hp = hp;
+        this.level = level;
+        this.food = food;
+        this.sound = sound;
+        this.attacks = Arrays.asList("Vine Whip", "Razor Leaf", "Solar Beam", "Leaf Storm");
+    }
+
+    @Override
+    public String getName() {
+        return name;
+    }
+
+    @Override
+    public int getHp() {
+        return hp;
+    }
+
+    @Override
+    public int getLevel() {
+        return level;
+    }
+
+    @Override
+    public String getFood() {
+        return food;
+    }
+
+    @Override
+    public String getSound() {
+        return sound;
+    }
+
+    @Override
+    public String getType() {
+        return "Grass";
+    }
+
+    @Override
+    public List<String> getAttacks() {
+        return attacks;
+    }
+
+    @Override
+    public void setHp(int damage) {
```

Comment:
Deze methode had je beter "subtractDamage" ofzo kunnen noemen. Je noemt het nu "setHp", maar dat is niet wat het doet.